### ISSUE TITLE
APIs doesn't show up anymore on Kong-Dashboard 3.4.0

### ISSUE THREAD

**Issue by gottfrois** [PGBI/kong-dashboard#178]
## Issue summary

Just upgraded to latest kong-dashboard 3.4.0 and the (deprecated) APIs tab is empty now.
https://d.pr/i/CZQuQS

I don't see any errors in the logs:

```
Connecting to Kong on http://kong-kong-admin.kong.svc.cluster.local:8001 ...
Connected to Kong on http://kong-kong-admin.kong.svc.cluster.local:8001.
Kong version is 0.13.1
Starting Kong Dashboard on port 8080
Kong Dashboard has started on port 8080
```

## Your environment

* Kong version = 0.13.1
* Kong Dashboard version = 3.4.0
* Docker image used = pgbi/kong-dashboard:latest

### Current behavior

APIs don't show up any more in the APIs tab.

### Expected behavior

APIs should show up.

### Steps to reproduce

1. Add APIs with previous kong-dashboard version
2. Upgrade to latest kong-dashboard

## Good to have

Listing my APIs using the same internal kong admin url from docker shell:

```
url http://kong-kong-admin.kong.svc.cluster.local:8001/apis
{"total":4,"data":[{"created_at":153 ...
```

**Comment by gottfrois** [comment]
`APIs.view.html` seems to fail with error code `404` https://d.pr/i/yghXni

**Comment by PGBI** [comment]
Ohhhh good catch @gottfrois . I'm working on macOS, which isn't case sensitive and the correct filename is apis.view.html.

**Comment by PGBI** [comment]
This should be fixed with kong-dashboard v3.4.1. Thanks for the report.
